### PR TITLE
add student loan fields, fix #2

### LINKED ISSUE #2: campaign id and key shouldn't be "lead." variables
They'll be configured during flow setup, not present on each lead, so`ckm_campaign_id` and `ckm_key` shouldn't be prefixed with `lead.`.

### LINKED ISSUE #4: add support for fields in the student loan vertical

### PR DESCRIPTION
**Pull request by jgrayson676**
also fixes #4

### CODE REVIEW

**Comment by cgrayson** [comment]
As discussed separately, re-consider using `mortgage.loan` fields for the student loan data.

**Comment by alexkwolfe** [comment]
I'm curious what the discussion was around using mortgage fields for student loans this way. I think mortgage fields are also being used for solar leads that have nothing to do with mortgage. The reuse of the fields in his way makes me nervous and there's a broader discussion going on about how to best build out our catalog of fields.

File: src/outbound.coffee [hunk 1]
```diff
@@ -9,53 +9,62 @@ moment = require('moment')
...
-    smoker: vars.lead.insurance?.tobacco_use?.valueOf()
-    existing_conditions: vars.lead.existing_conditions
-    expectant_parent: vars.lead.expectant_parent
-    life_event: vars.lead.life_event
-    denied_health: vars.lead.insurance?.previously_denied?.valueOf()
-    dob: if vars.lead.dob then moment(vars.lead.dob.valueOf()).format('MM/DD/YYYY')
-    user_agent: vars.lead.user_agent
+    gender: lead.gender?.valueOf()
+    income: lead.monthly_salary?.valueOf()
+    heigh_ft: lead.insurance?.height_ft?.valueOf()
```
Comment (cgrayson) [inline src/outbound.coffee]: Typo here.

**Comment by cgrayson** [comment]
:+1:

**Pull request merged**